packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/mouseUtils/handleMouseDownEvent.ts

```diff
@@ -53,65 +53,99 @@ export function handleMouseDownEvent(
                     selection?.setBaseAndExtent(state.firstTarget, 0, state.lastTarget, 0);
                     selectTable(editor, state);
                 }
-
                 return;
             }
         }
     }
-    if (which == LEFT_CLICK && !shiftKey) {
-        clearState(state, editor);
+    if (which == LEFT_CLICK) {
+        // Triple clicking a cell will select that cell
+        if (detail >= 3) {
+            editor.runAsync(editor => {
+                const sel = editor.getDocument().defaultView?.getSelection();
+                const first = getCellAtCursor(editor, sel?.anchorNode);
+                const firstTable = getTableAtCursor(editor, first);
+                const targetTable = getTableAtCursor(editor, first);
+                if (firstTable! == targetTable! && safeInstanceOf(first, 'HTMLTableCellElement')) {
```

Comment:
In what case firstTable != targetTable? From line 66 and 67, right side of "=" are the same, so they should be always the same